### PR TITLE
Updated the DM_API_AUTH_TOKENS environment variable

### PR DESCRIPTION
**Pull request by MuriloDalRi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ By default the API runs on port 5000. Calls to the API require a valid bearer
 token. Tokens to be accepted can be set using the DM_AUTH_TOKENS environment
 variable (a colon-separated list), e.g.:
 
-```export DM_API_AUTH_TOKENS=myToken1:myToken2```
+```export DM_API_AUTH_TOKENS=myToken```
```
Comment (TheDoubleK) [inline README.md]: It's quite nice to show the colon-separated list still.

So could be `myToken:anotherToken` or something like that?

Comment (MuriloDalRi) [inline README.md]: I had a look in the config.py, and in development the token is set to "myToken", so the app accepts a colon-separated list of tokens in production?

Comment (TheDoubleK) [inline README.md]: Yes, in production of anywhere - `myToken` is completely arbitrary default for development, you can specify anything you like. Multiple allowed tokens can be specified, and it's nice to keep the README documentation to reflect this I think.

Comment (TheDoubleK) [inline README.md]: In production (and other life environments) we generally have three active tokens - one for dev, one for Jenkins and one for the front-end apps.

**Pull request closed without merging**